app/(auth)/login/page.tsx

```
@@ -0,0 +1,112 @@
+        
+        if (!email.match(/^\S+@\S+\.\S+$/)) {
+            setEmailError(true);
+            return;
+        }
+
+        try {
+            // Add your login logic here
+            // If login fails, set loginError to true
+        } catch {
+            setLoginError(true);
+        }
```

Biome

[error] This code is unreachable



(lint/correctness/noUnreachable)